src/auth.ts

```diff
@@ -30,6 +33,42 @@ export class Auth {
     return false;
   }
 
+  /**
+   * Try to read ~/.neru-cli and extract credentials
+   * Fails silently on error
+   */
+   public static async loginWithNeru(): Promise<void> {
```

Comment:
can we name it `loginWithNeruIfAvailable` or something like this so its clearer that it could fail at the call site?